src/ui/actions/UnlabelAllTasksAction.h

```diff
@@ -0,0 +1,22 @@
+//
+// Created by Anton O. on 1/17/22.
+//
+
+#ifndef TASKMANAGER_SRC_UI_ACTIONS_UNLABELALLTASKSACTION_H
+#define TASKMANAGER_SRC_UI_ACTIONS_UNLABELALLTASKSACTION_H
+
+#include "Action.h"
+
+class UnlabelAllTasksAction : public Action {
```

Comment:
Confusing class name

Reply:
renamed UnlabelAllTasksAction to ClearAllLabelsOfTaskAction
UnlabelTaskAction to ClearLabelOfTaskAction
GetTasksToShowLabelsAction to GetTaskToShowLabelsAction